### PR TITLE
[TECHNICAL-SUPPORT] LPS-70087

### PR DESCRIPTION
**Pull request by jonathanmccann**
/cc @mbowerman

### CODE REVIEW

**Comment by mbowerman** [comment]
These OrganizationIndexerTest failures are all happening in BChan's latest pull requests;

https://github.com/brianchandotcom/liferay-portal/pull/46256
https://github.com/brianchandotcom/liferay-portal/pull/46255
https://github.com/brianchandotcom/liferay-portal/pull/46254

etc.

**Comment by mbowerman** [comment]
ci:retest

File: portal-kernel/src/com/liferay/portal/kernel/workflow/WorkflowTaskManager.java [hunk 1]
```diff
@@ -120,6 +120,9 @@ public int getWorkflowTaskCountByWorkflowInstance(
 			OrderByComparator<WorkflowTask> orderByComparator)
 		throws WorkflowException;
 
+	public boolean hasOtherAssignees(long workflowTaskInstanceId, long userId)
```
Comment (marcellustavares) [inline portal-kernel/src/com/liferay/portal/kernel/workflow/WorkflowTaskManager.java]: @mbowerman, is this really needed? Can we avoid exposing this method?

**Comment by marcellustavares** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: portal-kernel/src/com/liferay/portal/kernel/workflow/WorkflowTaskManager.java (hunk 1 above)
Comment (mbowerman) [inline portal-kernel/src/com/liferay/portal/kernel/workflow/WorkflowTaskManager.java]: @marcellustavares If we want to make the hasOtherAssignees method more efficient, then I don't see any other way to do it other than creating this new method. If you don't think the benefit to performance is worth the cost of creating a new method, then let me know and we can just close this ticket as Won't Fix.

**Comment by marcellustavares** [comment]
Pull request submitted to https://github.com/inacionery/liferay-portal/pull/118. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**